NightscoutUploadKit/NightscoutUploader.swift

```diff
@@ -122,20 +122,6 @@ public class NightscoutUploader {
         uploadToNS(treatments, endpoint: defaultNightscoutTreatmentPath, completion: completionHandler)
     }
 
-    public func getPumpStatusFromMySentryPumpStatus(status: MySentryPumpStatusMessageBody) throws -> PumpStatus {
```

Comment:
If you're OK with MinimedKit remaining a dependency on Nightscout upload it, then I'd make it a convenience initializer on PumpStatus. 

I didn't see it moved to DDM, sorry - I think that's a perfectly reasonable step in a path to allow NightscoutUloadKit to stand alone